src/components/CreateCoursePage/CreateCourseForm.jsx

```diff
@@ -109,6 +120,17 @@ class BaseCreateCourseForm extends React.Component {
             label={<FieldLabel text="Organization" required />}
             required
           />
+          <Field
+            name="source"
+            component={RenderSelectField}
+            options={this.processSources(sources)}
```

Comment:
This will make API call, correct? make the method return default as well, until the dropdown is required.

Reply:
This function receives data from an API call and transforms it into a list of dictionaries, where each dictionary has a "label" and "value" key. This is done so that the resulting list can be used as options for a select field.

dict format `{'label': '', 'value': 'key'}`